requirements.txt

```diff
@@ -1,3 +1,4 @@
+colorama
```

Comment:
colorama likely not needed. (still including it as color is needed for logging and might be required later)